modules/rosa-planning-considerations.adoc

```diff
@@ -71,7 +71,7 @@ The maximum number of compute nodes on ROSA is 180.
 
 The SRE team might scale the control plane and infrastructure nodes for additional reasons, for example to manage an increase in resource consumption on the nodes.
 
-When scaling is applied, the customer is notified through a service log entry.
+When scaling is applied, the customer is notified through a service log entry. For more information about the service log, see _Accessing the service logs for ROSA clusters_.
```

Comment:
Question, sir: Is the "," needed here?

Reply:
In my view the comma in this sentence is ok. There are many instances of "For more information, see..." type lines in the docs.